google/cloud/language/v1beta1/language_gapic.yaml

```diff
@@ -3,7 +3,7 @@ language_settings:
   java:
     package_name: com.google.cloud.language.spi.v1beta1
   python:
-    package_name: google.cloud.language.v1beta1
```

Comment:
https://github.com/GoogleCloudPlatform/google-cloud-python/issues/2258

Basically, an element in a Python path can be a "namespace" or it can define "members", but not both. Our current scheme of putting hand-written code in `google.cloud.{api}` and auto-gen code in `google.cloud.{api}.{version}` means that `google.cloud.{api}` is a namespace, which interferes with the google-cloud-python's team use of members to improve developer experience.

I can forward you an email with the full discussion, and you can also take a look at the Python section of the CCL Namespacing PRD to see the schema we're moving toward.